docs/kickstart-docs.rst

```diff
@@ -130,6 +130,13 @@ You could use an entry similar to the following:
 
 ``ignoredisk --drives=/dev/disk/by-path/pci-0000:00:05.0-scsi-*``
 
+However, the ``*`` pattern will fail the installation in case no device could
+be matched. For ``clearpart`` and ``ignoredisk`` commands specifically you
+can also use ``|`` to solve this issue. For example to match ``vda`` and/or
+``hda`` you can use ``vda|hda`` which would match ``vda`` or ``hda`` or both.
+If nor ``vda`` or ``hda`` disks are not available the installation will fail.
```

Comment:
Rewording:
But if neither ``vda`` or ``hda`` are available the installation will fail.
If you want to match any ``vd`` or ``hd`` disk you could combine ``*`` and ``|`` to match any drive of either type: ``vd*|hd*``